docs/topics/api/signing.rst

```diff
@@ -55,29 +74,55 @@ longer. Once review is complete then the ``reviewed`` property
 will be set and you can check the results with the ``passed_review``
 property.
 
-To make the request using curl::
+To make the request using curl:
 
-    curl https://addons.mozilla.org/en-US/firefox/api/v3/addons/my-addon/versions/1.0/
-         -H 'Authorization: JWT <jwt-token>'
+.. http:get:: /en-US/firefox/api/v3/addons/(add-on-guid)/versions/(version)/
 
-Here's a full example of a check status response::
+    **Request:**
+
+    .. sourcecode:: bash
+
+        curl https://addons.mozilla.org/en-US/firefox/api/v3/addons/my-addon/versions/1.0/
+            -H 'Authorization: JWT <jwt-token>'
+
+    :param addon-guid: the GUID for the add-on.
+    :param version: the version of the add-on.
+
+    **Response:**
+
+    .. code-block:: json
 
-    {
-        "active": true,
-        "files": [
             {
-                "download_url": "https://addons.mozilla.org/firefox/downloads/file/100/unlisted_wat-0.0.0-fx+an.xpi?src=api",
-                "signed": true
+                "active": true,
+                "files": [
+                    {
+                        "download_url": "https://addons.mozilla.org/firefox/downloads/file/100/unlisted_wat-0.0.0-fx+an.xpi?src=api",
```

Comment:
I probably should have sanitized the filename and guid so they weren't so silly.